Gordon360/Models/ViewModels/ClockInViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class ClockInViewModel
+    {
+        //is true if user has clocked in and false if user is going to clock out
+        public bool currentState { get; set; }
+
+        //time of when when user clocked in
+        public DateTime timestamp { get; set; }
+    }
+}
```

Comment:
Got hopefully you mean like literally pressing enter. otherwise I'm not sure what you mean by a new line. Dr. Tuck has made a comment similar to this one and I was just as confused haha.